src/PrestaShopBundle/Bridge/Helper/Listing/HelperListConfigurator.php

```diff
@@ -63,23 +63,23 @@ public function setHelperDisplay(
         HelperListConfiguration $helperListConfiguration,
         HelperList $helper
     ): void {
-        $breadcrumbs = $this->breadcrumbsAndTitleHydrator->getBreadcrumbs($helperListConfiguration->id);
+        $breadcrumbs = $this->breadcrumbsAndTitleHydrator->getBreadcrumbs($helperListConfiguration->getTabId());
```

Comment:
Still not sure about the use of this class but it's not the point of this PR 😅

Reply:
maybe we should just move it all to the same factory and remove this configurator? Im not really sure what was the aim here initially, but we never call this class anywhere outside of factory

Reply:
Yeah that's what I had in mind as well, the weirdest thing is that it's also based on a hydrator which is also used somewhere else, I guess the idea is also to avoid duplicating code but then it means the data generated by the `breadcrumbsAndTitleHydrator` should already be accessible in another configuration no?

Although it's not blocking for this one.